src/py-opentimelineio/opentimelineio/adapters/otio_json.py

```diff
@@ -4,11 +4,13 @@
 """This adapter lets you read and write native .otio files"""
 
 from .. import (
-    core
+    core,
+    versioning
 )
 
+import os
 
-# @TODO: Implement out of process plugins that hand around JSON
+DEFAULT_VERSION_ENVVAR = "OTIO_DEFAULT_TARGET_VERSION_FAMILY_LABEL"
```

Comment:
You might want to add this environment variable to https://github.com/AcademySoftwareFoundation/OpenTimelineIO/blob/main/docs/tutorials/otio-env-variables.md.